ChocolateyPackage/textanalysistoolnet.portable.nuspec

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>textanalysistoolnet.portable</id>
+    <version>2018.11.20</version>
+    <owners>bcurran3</owners>
+    <packageSourceUrl>https://github.com/TextAnalysisTool/Releases/tree/master/ChocolateyPackage</packageSourceUrl>
+    <title>TextAnalysisTool.NET (Portable)</title>
+    <authors>David Anson</authors>
```

Comment:
Does it allow listing multiple people?

Reply:
Looks like it does, let me try adding.

Reply:
Can we list the other folks at the top of https://textanalysistool.github.io/